### DISCUSSION TITLE
Welcome to whatsapp-webhook Discussions!

### DISCUSSION THREAD

**Discussion by junedkhatri31** [whatsapp-webhook/whatsapp-webhook#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by niyamvora** [comment]
Hi, 

I am in urgent need for the whatsapp web setup.

I will pay you as well, all you have to do is set me up whatsapp chat on my own number. I will provide you with whatever credentials for whatspp api, and you can just help me start this thing instantly. 

If you can do it within few hours since you already have made this app and all you would require would be to enter the credentials, just drop me email at **niyamvora@gmail.com** and as soon as set up is complete i will transfer you the funds .

Also if you need some advance do let me know as well.

Request you to connect with me as soon as possible.

**Reply by junedkhatri31** [reply]
Hi Niyam,
Apologies for the delay. I was on vacation. I am happy to get in touch with you.

**Reply by niyamvora** [reply]
Hi Juned,

I am waiting for your call since 3 days. i have also dropped you email with my number 2-3 times.

Please connect with me as soon as possible.

**Comment by abrarapatel** [comment]
Hi! It's a great project specially now. As it is time of ai and there are lot of businesses who need ai chatbot in apps like whatsapp. I was wondering if this project is getting continued?

Tell me if anyone is working on this project. I would like to join.

Thank you.
